.github/CODEOWNERS

```diff
@@ -66,22 +66,22 @@
 /doc/using @fricklerhandwerk
 
 # NixOS Internals
-/nixos/default.nix          @nbp @infinisil
-/nixos/lib/from-env.nix     @nbp @infinisil
-/nixos/lib/eval-config.nix  @nbp @infinisil
-/nixos/doc/manual/configuration/abstractions.xml      @nbp
-/nixos/doc/manual/configuration/config-file.xml       @nbp
-/nixos/doc/manual/configuration/config-syntax.xml     @nbp
-/nixos/doc/manual/configuration/modularity.xml        @nbp
-/nixos/doc/manual/development/assertions.xml          @nbp
-/nixos/doc/manual/development/meta-attributes.xml     @nbp
-/nixos/doc/manual/development/option-declarations.xml @nbp
-/nixos/doc/manual/development/option-def.xml          @nbp
-/nixos/doc/manual/development/option-types.xml        @nbp
-/nixos/doc/manual/development/replace-modules.xml     @nbp
-/nixos/doc/manual/development/writing-modules.xml     @nbp
-/nixos/doc/manual/man-nixos-option.xml                @nbp
-/nixos/modules/installer/tools/nixos-option.sh        @nbp
+/nixos/default.nix                                    @infinisil
+/nixos/lib/from-env.nix                               @infinisil
+/nixos/lib/eval-config.nix                            @infinisil
+/nixos/doc/manual/configuration/abstractions.xml
+/nixos/doc/manual/configuration/config-file.xml
+/nixos/doc/manual/configuration/config-syntax.xml
+/nixos/doc/manual/configuration/modularity.xml
+/nixos/doc/manual/development/assertions.xml
+/nixos/doc/manual/development/meta-attributes.xml
+/nixos/doc/manual/development/option-declarations.xml
+/nixos/doc/manual/development/option-def.xml
+/nixos/doc/manual/development/option-types.xml
+/nixos/doc/manual/development/replace-modules.xml
+/nixos/doc/manual/development/writing-modules.xml
+/nixos/doc/manual/man-nixos-option.xml
+/nixos/modules/installer/tools/nixos-option.sh
```

Comment:
Should we just drop these lines?

Reply:
Technically they do not do anything, so we could.

Reply:
I just dropped them. Thanks for the reminder.